spring/src/main/java/io/seata/spring/annotation/GlobalTransactionalInterceptor.java

```diff
@@ -160,9 +158,17 @@ public TransactionInfo getTransactionInfo() {
     }
 
     private String formatMethod(Method method) {
-        String paramTypes = Arrays.stream(method.getParameterTypes())
-                .map(Class::getName)
-                .collect(Collectors.joining(", ", "(", ")"));
-        return method.getName() + paramTypes;
+        StringBuilder sb = new StringBuilder(method.getName())
+                .append("(");
+
+        Class<?>[] params = method.getParameterTypes();
+        int in = 0;
+        for (Class<?> clazz : params) {
+            sb.append(clazz.getName());
+            if (++in < params.length) {
+                sb.append(", ");
+            }
+        }
+        return sb.append(")").toString();
```

Comment:
@yangfuhai Is the performance difference from the usage of + and append?

Reply:
the performance is 10 times higher than before. this is particularly important in high concurrency.

Reply:
Collectors.joining is implemented via StringBuilder.append, so the difference comes from method.getName() + paramTypes and StringBuilder(method.getName()).append?

Reply:
the reason for the performance gap maybe is from Arrays.stream.map vs for-loop。

Reply:
recommended to use StringJoiner